pytket/pytket/qasm/qasm.py

```diff
@@ -1042,7 +1043,17 @@ def circuit_to_qasm_io(
         elif header == "hqslib1" and optype == OpType.ZZPhase:
             # special handling for zzphase
             opstr = "RZZ"
-            params = op.params
+            param = op.params[0]
+            # as op.params returns reduced parameters, we can assume
```

Comment:
How sure are we about this? Should we do an assert?

Reply:
It's certainly the case at the moment, if it does change then the test in qasm_test.py will fail so it can be corrected.

Reply:
Okay, then keep it as it is